torch/nn/_functions/rnn.py

```diff
@@ -18,8 +20,15 @@ def RNNTanhCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
 
 
 def LSTMCell(input, hidden, w_ih, w_hh, b_ih=None, b_hh=None):
+    if input.is_cuda:
+        igates = F.linear(input, w_ih, None)
+        hgates = F.linear(hidden[0], w_hh, None)
+        state = fusedBackend.LSTMFused()
+        return b_ih and state(igates, hgates, hidden[1], b_ih, b_hh) or state(igates, hgates, hidden[1])
```

Comment:
Took it from... https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L431-L433

Will fix both.